modbus/README.md

```diff
@@ -0,0 +1,30 @@
+## Modbus Monitoring Template
```

Comment:
This should be "Island Pulse Monitoring Template" since it's specific to that setup, and not "Modbus" in general